<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_ui</class>
 <widget class="QDialog" name="client_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>559</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Book Storage</string>
  </property>
  <widget class="QTreeView" name="book_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>561</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>521</width>
      <height>501</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>20</width>
      <height>501</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>verticalScrollBar</zorder>
   <zorder>textBrowser</zorder>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>19</y>
     <width>191</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>174</width>
      <height>515</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Options panel: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_create_book">
         <property name="text">
          <string>New Book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_new_season">
         <property name="text">
          <string>New Season</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_book_name">
         <property name="text">
          <string>Change Book Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_edit">
         <property name="text">
          <string>Edit line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_edit_writers">
         <property name="text">
          <string>Edit Writers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Export_changes">
         <property name="text">
          <string>Export changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_latest_version">
         <property name="text">
          <string>Get Latest Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_show_edits">
         <property name="text">
          <string>Show Season content</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_version_content">
         <property name="text">
          <string>Version content</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_back_version">
         <property name="text">
          <string>Back to version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_publish">
         <property name="text">
          <string>Publicate Book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_published">
         <property name="text">
          <string>Published books</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_users_requests">
         <property name="text">
          <string>Show users requests</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_grant">
         <property name="text">
          <string>Grant a request</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Sync</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3"/>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>941</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>book_list</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
